### DISCUSSION TITLE
Invalid config path

### DISCUSSION THREAD

**Discussion by JesusMF23** [NVIDIA-NeMo/Guardrails#122] [Q&A] [answered]
Hi guys,

I am trying to implement moderation using guardrails so the bot verifies the response before returning it. For that, I have defined a context (is a json object stringified). I have used the [output_moderation_v2.py](https://github.com/NVIDIA/NeMo-Guardrails/blob/main/nemoguardrails/actions/output_moderation_v2.py) on my code as follows:

```
async def output_moderation_v2(
    llm_task_manager: LLMTaskManager,
    context: Optional[dict] = {"<stringified_json>"},
    llm: Optional[BaseLLM] = AzureChatOpenAI(<config>),
```

I have the following structure on my project:

```
-rails
--moderation.co
--general.co
--config.yaml
-guardrails.py
-output_moderation.py
```

my `guardrails.py` is my main file and from here I am calling as follows: 
```
config = RailsConfig.from_path("rails")
app = LLMRails(config, llm=llm, verbose=True)
```

However, I am getting the following error: `  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/nemoguardrails/rails/llm/config.py", line 360, in from_path
    raise ValueError(f"Invalid config path {config_path}.")
ValueError: Invalid config path rails.`

Basically following this documentation: [moderation_rail](https://github.com/NVIDIA/NeMo-Guardrails/blob/main/examples/moderation_rail/README.md#api)

Does anybody knows why is this happening?

**Comment by JesusMF23** [comment, marked as answer]
My bad, I am implementing this in google cloud functions, therefore the path is not just `"rails"`, as my code is within the function files for it to be available in google cloud, the path is `"functions/rails"`
